<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1518</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
  background-color: rgb(238, 238, 238);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
  background-color: rgb(238, 238, 238);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget{
border: 0;
font-family: monospace;
}
QTabWidget::tab-bar {
    left:340px;


}

QTabWidget::tab-bar:top {
top: 1px;
}

QTabWidget::tab-bar:bottom {
bottom: 1px;
}

QTabWidget::tab-bar:left {
right: 20px;
}

QTabWidget::tab-bar:right {
left: 20px;
}

QTabBar::tab {
border: 0.5px solid white;
width: 400;
font-size: 20px;
height:30;


}

QTabBar::tab:selected {
background: white;
color: rgb(5, 119, 119);
}

QTabBar::tab:!selected {
background: rgb(5, 119, 119);
color: white;

}

QTabBar::tab:!selected:hover {
background: #999;
}

QTabBar::tab:top:!selected {
margin-top: 3px;
}

QTabBar::tab:bottom:!selected {
margin-bottom: 3px;
}

QTabBar::tab:top, QTabBar::tab:bottom {
min-width: 8ex;
margin-right: -1px;
padding: 5px 10px 5px 10px;
}

QTabBar::tab:top:selected {
border-bottom-color: none;
}

QTabBar::tab:bottom:selected {
border-top-color: none;
}

QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one {
margin-right: 0;
}

QTabBar::tab:left:!selected {
margin-right: 3px;
}

QTabBar::tab:right:!selected {
margin-left: 3px;
}

QTabBar::tab:left, QTabBar::tab:right {
min-height: 8ex;
margin-bottom: -1px;
padding: 10px 5px 10px 5px;
}

QTabBar::tab:left:selected {
border-left-color: none;
}

QTabBar::tab:right:selected {
border-right-color: none;
}

QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one {
margin-bottom: 0;
}


</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="FeatureExtraction">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Feature Extraction</string>
       </attribute>
       <widget class="QPushButton" name="uploadButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>60</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-radius: 20px;
 
    background-color: rgb(5, 119, 119);
    border: none;
    color: white;
    font-size: 20px;
}

QPushButton::hover {
background-color: white;
color: rgb(5, 119, 119);
}</string>
        </property>
        <property name="text">
         <string>Upload Image</string>
        </property>
       </widget>
       <widget class="QLabel" name="HarrisLabel">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>130</y>
          <width>350</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font-size: 20px;
    font-family: monospace;
}</string>
        </property>
        <property name="text">
         <string>Harris </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="HarrisImage">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>190</y>
          <width>350</width>
          <height>450</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
border: 10px;
border-radius: 22px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="uploadImage">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>350</width>
          <height>450</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
border: 10px;
border-radius: 22px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="uploadImageLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>350</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font-size: 20px;
    font-family: monospace;
}</string>
        </property>
        <property name="text">
         <string>Uploaded Image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="SiftLabel">
        <property name="geometry">
         <rect>
          <x>1130</x>
          <y>130</y>
          <width>350</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font-size: 20px;
    font-family: monospace;
}</string>
        </property>
        <property name="text">
         <string>Sift </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="SiftImage">
        <property name="geometry">
         <rect>
          <x>1130</x>
          <y>190</y>
          <width>350</width>
          <height>450</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
border: 10px;
border-radius: 22px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="EigenImage">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>190</y>
          <width>350</width>
          <height>450</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
border: 10px;
border-radius: 22px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="EigenLabel">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>130</y>
          <width>350</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font-size: 20px;
    font-family: monospace;
}</string>
        </property>
        <property name="text">
         <string>Eigen</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="FeatureMatching">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Feature Matching</string>
       </attribute>
       <widget class="QPushButton" name="sift1Button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-radius: 20px;
 
    background-color: rgb(5, 119, 119);
    border: none;
    color: white;
    font-size: 20px;
}

QPushButton::hover {
background-color: white;
color: rgb(5, 119, 119);
}</string>
        </property>
        <property name="text">
         <string>Upload Image 1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sift2Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>450</y>
          <width>180</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-radius: 20px;
 
    background-color: rgb(5, 119, 119);
    border: none;
    color: white;
    font-size: 20px;
}

QPushButton::hover {
background-color: white;
color: rgb(5, 119, 119);
}</string>
        </property>
        <property name="text">
         <string>upload Image 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="uploadsift1">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>400</width>
          <height>350</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="uploadsift2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>370</y>
          <width>400</width>
          <height>350</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="resultLabel">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>370</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font-size: 20px;
    font-family: monospace;
}</string>
        </property>
        <property name="text">
         <string>Matching by Normalized Cross Correlation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="ssdLabel">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>10</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	font-size: 20px;
    font-family: monospace;
}</string>
        </property>
        <property name="text">
         <string>Matching by SSD</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="ncc">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>400</y>
          <width>400</width>
          <height>330</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ssd">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>40</y>
          <width>400</width>
          <height>330</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
